physics/src/modules/FluxCalculationModule/include/FiniteElementFluxes.hpp

```diff
@@ -122,7 +124,7 @@ class FiniteElementFluxes : public IFluxCalculation, public Configured<FiniteEle
     void calculateAtmos(size_t i, const TimestepTime& tst);
 
     static double dragOcean_q;
-    static double dragOcean_m(double windSpeed);
+    static inline double dragOcean_m(double windSpeed);
```

Comment:
When defining a function inline, the function body needs to be present, otherwise the compiler cannot inline the function. Or rather, there is no possibility that the function will be defined multiple times, due to its presence in the header.
So remove the `inline` or move the function body to the header.